ubuntu_dockerfile

```diff
@@ -203,7 +203,7 @@ ENTRYPOINT ["dumb-init"]
 
 # Install packages needed for Google Chrome
 RUN apt-get update && \
-   apt-get install -y fonts-liberation libappindicator3-1 libgtk-3-0 libxss1 lsb-release xdg-utils
+   apt-get install -y fonts-liberation libappindicator3-1 libgtk-3-0 libxss1 libgbm1 lsb-release xdg-utils
```

Comment:
is this an auto change?

Reply:
No, but the installation failed for me at this point on multiple machines, and for `@jassi-singh` in the gitter chat.

This is the fix we both found, should it be a part of a different PR?

Reply:
Nope, it's fine. Thanks for the explanation and for fixing the issue! :)